truffle-config.js

```diff
@@ -38,19 +20,14 @@ module.exports = {
   },
   compilers: {
     solc: {
-      version: "0.8.4",
+      version: statics.compilers[process.env.WITNET_EVM_REALM || "default"].version,
       settings: {
         optimizer: {
           enabled: true,
           runs: 200,
         },
       },
+      evmVersion: statics.compilers[process.env.WITNET_EVM_REALM || "default"].evmVersion,
```

Comment:
Even more powerful:
```suggestion
      settings: {
        optimizer: {
          enabled: true,
          runs: 200,
        },
      },
      ...statics.compilers[process.env.WITNET_EVM_REALM || "default"]
```

Reply:
I found this article https://flaviocopes.com/how-to-merge-objects-javascript/, referring to Lodash, whose `merge()` would be even more powerfull still, as it would enable the intended functionality: take all parameters set for "default" and then set/overwrite parameters set for actual `process.env.WITNET_EVM_REALM`.